websites/N/Ngewibu/presence.ts

```diff
@@ -0,0 +1,26 @@
+const presence = new Presence({
+		clientId: "1104875968134397963",
+	}),
+	presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/ZmylEPM.png",
+		startTimestamp: Math.floor(Date.now() / 1000),
+	};
+
+presence.on("UpdateData", async () => {
+	if (document.location.pathname.startsWith("/play")) {
+		presenceData.details = "Watching anime";
+		presenceData.state = document
+			.querySelector("title")
+			.textContent.replace("Nonton ", "")
+			.replace(/- NgewibuTV/gi, "");
+		presenceData.buttons = [
+			{ label: "View Anime", url: document.location.href },
+		];
+	} else {
+		presenceData.details = "Viewing index";
+		presenceData.buttons = [
+			{ label: "Visit NgewibuTV", url: document.location.href },
+		];
+	}
+	presence.setActivity(presenceData);
+});
```

Comment:
Low quality presence

Reply:
Not much to do and no executable paths in https://ngewibu.tv/sitemap.xml
There is a genre, example https://ngewibu.tv/index/action if i adding this, is that still counted as low presence or not

Reply:
Add videos + timestamps
What you're browsing
Etc

Reply:
```ts
// Title for Details
document.querySelector('[class="mb-1  text-truncate"]')?.textContent
```
```ts
// Episode (For state)
document.querySelector('[class*="currentPlay bg-success"]')?.textContent.replace("E", "Episode")
```

```ts
// Thumbnail of the episode (for largeImageKey)
`https://${location.hostname}${document.querySelector('[class="img-fluid"]')?.getAttribute("src").replace(".webp", ".png")}`
```

```ts
// Video detection
const video = document.querySelector<HTML("video")
```

Reply:
Thanks for clarifying, I actually don't show off a much with DC presence, thus the design is a bit vague, I'll update this tonight.

Also what is video detection used for?

Reply:
> Thanks for clarifying, I actually don't show off a much with DC presence, thus the design is a bit vague, I'll update this tonight.
> 
> Also what is video detection used for?

Video detection is used for knowing the video timestamp & to know when it's paused